packages/plugins/solc/src/lib/Compiler.js

```diff
@@ -48,7 +48,7 @@ function compileSolc(embark, contractFiles, contractDirectories, options, callba
   }
 
   const logger = embark.logger;
-  const outputBinary = embark.pluginConfig.outputBinary;
+  const outputBinary = embark.config.pluginConfig.outputBinary;
```

Comment:
This  could used to rely on `embark.pluginConfig` and that works at runtime, because the thing we call `embark` here is actually an instance of `Plugin`. However, in our testing environment we actually inject something that looks more like an `Embark` than a `Plugin`. Therefore, `pluginConfig` doesn't exist there right on the `Embark` object (because it really doesn't, it only does on `Plugin`). 

But both, `Embark` and `Plugin` have a reference to `Config` which also exposes `pluginConfig`. 

Long story short, for consistency reasons and to make this code work in tests, I read from `embark.config` here instead.